schematic/store/synapse.py

```diff
@@ -1861,7 +1871,17 @@ async def add_annotations_to_entities_files(
             # Merge dataframes to add entityIds
             manifest = manifest.merge(
                 file_df, how="left", on="Filename", suffixes=["_x", None]
-            ).drop("entityId_x", axis=1)
+            )
+
+            # drop the duplicate entity column with NA values
+            col_to_drop = "entityId_x"
+            if manifest.entityId.isnull().all():
+                col_to_drop = "entityId"
```

Comment:
Will there ever be a time when there is a mixture of NA and non-NA values within this column?

Reply:
From my understanding of how `pd.df.merge` works, no. It should always be one or the other completely blank

Reply:
I am resolving this conversation to let this PR gets merged. But we are not dealing with the "merge" issue in this PR any more.